### DISCUSSION TITLE
Funny layer duration... I guess something may be missconfigured with me?

### DISCUSSION THREAD

**Discussion by rbid** [j7126/OctoPrint-Dashboard#348] [Q&A] [answered]
Hi,
I installed the **OctoPrint-Dashboard**, because it describes that it does exactly what I need, and before I complain, I want to say thanks for the great work you have done.

I have installed the plugin via the plugin manager in OctoPrint(v 1.8.1), running from OctoPi (v 0.18.0) on a Raspberry pi 4 with 4G mem, using Python 3.7.3.

After printing starts, I start to see funny numbers with the Layer gauge .. See (1) from the attached pic.
The Layer Durations (2) is empty and does not change when configuring it.
The bottom layer summary stays at zero, even if displays the correct number of layers.

The screenshot was taken when the print was at layer 97.
![Region](https://user-images.githubusercontent.com/2997037/172818386-2006df31-d493-4c91-ba71-df7865adfdc4.png)
Printing using a Prusa MINI+ connected via USB to Octoprint.

Any hint what is wrongly configured?..

Greetings from the Galilee.

--- Ricky Marek.

**Comment by j7126** [comment, marked as answer]
Hi, there have been some issues with the gcode analysis recently. If I remember correctly some issues have currently been fixed in the latest development branch, but have not been released yet. Please check if you get any different results if you install the plugin using the development branch here `https://github.com/j7126/OctoPrint-Dashboard/archive/development.zip`. 

If that does not fix it please let me know and If you send me your gcode file I will be able to look into the issue. Also please let me know which slicer you are using.

**Reply by rbid** [reply]
Wow, that was fast.
I will take this branch and update you with the results during this weekend :)
Greetings from the Galilee.
--- Ricky Marek.

**Comment by rbid** [comment]
Thanks a lot, works very nice!
The development branch gives a nice result.
Greetings from the Galilee.
